include/pru/am572x.h

```diff
@@ -2,6 +2,34 @@
  *
  * pru/am335x.h -- PRU I/O registers definitions for PRU inside AM335x SoCs
  *
+ * Copyright (c) 2015-2019 Dimitar Dimitrov <dimitar@dinux.eu>
```

Comment:
Please do not duplicate the text license. I strongly suggest to leave BSD 2 clause and the SPDX header. The following should be enough:
```
 * pru/am572x.h -- PRU I/O registers definitions for PRU inside AM572x SoCs.
 *                          Based on pru/am335x.h
 *
 * Copyright (c) 2020 Vedant Paranjape
```

Reply:
Sure, I will make this change as you requested.

Reply:
Fixed in  c8b390497cd757a2839bff0fe27eb619c22c4049